west-european-politics.csl

```diff
@@ -0,0 +1,225 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" initialize="false" demote-non-dropping-particle="sort-only" default-locale="en-US" xmlns="http://purl.org/net/xbiblio/csl">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>West European Politics</title>
+    <id>http://www.zotero.org/styles/west-european-politics</id>
+    <link href="http://www.zotero.org/styles/west-european-politics" rel="self"/>
+    <link href="http://www.tandf.co.uk/journals/authors/style/reference/ref_fwep.pdf" rel="documentation"/>
+    <author>
+      <name>João Cancela</name>
+      <email>joaocancela@fcsh.unl.pt</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="political_science"/>
+    <issn>0140-2382</issn>
+    <eissn>1743-9655</eissn>
+    <summary>WEP</summary>
+    <updated>2015-12-30T10:11:08+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="editor">
+    <names variable="editor">
+      <name delimiter="," and="text"/>
+      <label form="short" strip-periods="true" prefix=" (" suffix=".)"/>
```

Comment:
delete `strip-periods="true"` and the period in the suffix -- they cancel each other out.